### PR TITLE
Update hyperspace docs how to run the tests

### PR DESCRIPTION
**Pull request by RustNinja**
- hyperspace/testsuite/README.md updated
- provided the detailed up to date instruction how to run IBC integration tests with two parachains.

### CODE REVIEW

**Review by kkast — APPROVED** [review]

File: hyperspace/testsuite/README.md [hunk 1]
```diff
@@ -93,5 +93,16 @@ Using the testsuite is straight forward and the following pseudocode describes t
...
+2. Build branch release-v0.9.x `cargo build -r --bin polkadot`
+3. build parachain-node in release `cargo build -r --bin parachain-node`
+4. run script:
+   - navigate to scripts folder in centauri repo: `cd scripts/polkadot-launch`
+   - `yarn install`
+   - `yarn dev`
+5. optional: regenerate subxt types [`more detailed instruction`](../../utils/subxt/codegen/README.md)
+   `cargo run --release -p codegen --bin codegen -- --path ./utils/subxt/generated/src/default`
+6. run the test
+   `cargo +nightly test -p hyperspace-testsuite --locked --release`
```
Comment (vmarkushin) [inline hyperspace/testsuite/README.md:108]: Let's use just `cargo run --bin codegen -- --path ./utils/subxt/generated/src/default` and `cargo test -p hyperspace-testsuite`

**Review by vmarkushin — COMMENTED** [review]
Gorgeous, thank you! Please address the previous comment and we can merge

File: hyperspace/testsuite/README.md (hunk 1 above)
Comment (blasrodri) [reply hyperspace/testsuite/README.md:108]: @RustNinja can we add this and merge?

Comment (RustNinja) [reply hyperspace/testsuite/README.md:108]: Yes. i waited once we merge testing optimisation that this changes took less time for CI to validate

Comment (RustNinja) [reply hyperspace/testsuite/README.md]: @blasrodri done.

**Review by blasrodri — APPROVED** [review]

**Pull request merged**